test/unit/io/sam_file/format_bam_test.cpp

```diff
@@ -179,6 +179,47 @@ struct sam_file_read<seqan3::format_bam> : public sam_file_data
         '\x66', '\x66', '\x66', '\x46', '\x40', '\x7A', '\x7A', '\x5A', '\x73', '\x74', '\x72', '\x00'
     };
 
+    std::string many_refs{[] ()
```

Comment:
The BAM would be multiple hundred lines long, that's why there it is generated.

The comments + BAM specification make it feasible to understand.

Reply:
We could have also generated a bam file and added it as file :D